docs/source/tracking.rst

```diff
@@ -421,6 +429,41 @@ Autologging captures the following information:
 |           |                        | `OneCycleScheduler`_ callbacks                           |               |                                                                                                                                                                       |
 +-----------+------------------------+----------------------------------------------------------+---------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Pytorch (experimental)
+--------------------------
+
+Call :py:func:`mlflow.pytorch.autolog` before your training code to enable automatic logging of metrics and parameters. See example usages with `Pytorch <https://github.com/mlflow/mlflow/tree/master/examples/pytorch>`_.
+
+Whether you are using torch 1.5 or 1.6, the respective metrics associated with ``EarlyStopping Callabacks`` and ``pytorch_lightning.trainer`` are automatically logged.
```

Comment:
@karthik-77 Please remove "Whether you are using torch 1.5 or 1.6" unless there is a limitation that things do not work on prior versions.

Reply:
@karthik-77 Mention that at present PyTorch autolog works with the Lightning training loop and include link for PTL